spec/internal/db/schema.rb

```
@@ -84,14 +84,31 @@
     t.timestamps
   end
 
-  create_table "coupon_codes", force: true do |t|
-    t.string "type"
-    t.string "coupon_code", null: false
-    t.boolean "redeemed", default: false
-    t.integer "shop_id"
-    t.integer "redeem_counter", default: 1
-    t.date "validity"
-    t.integer "free_days", default: 0
+  create_table "discount_codes", force: :cascade do |t|
```

RuboCop (1.76.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)